usr/src/common/zfs/zfs_prop.c

```diff
@@ -513,6 +521,21 @@ zfs_prop_user(const char *name)
 }
 
 /*
+ * Returns true if this is a valid quotas-type property.
+ */
+boolean_t
+zfs_prop_quotas(zfs_prop_t prop)
```

Comment:
not a big deal, but I think the singluar `zfs_prop_quota` is in keeping with the existing func names (e.g. `zfs_prop_userquota`, `zfs_prop_readonly`, etc).